test/cpp/jit/test_gpu.cpp

```diff
@@ -3915,6 +3915,47 @@ void testGPU_FusionZeroDimReduction() {
       aten_output.sub(output).abs().max());
 }
 
+void testGPU_FusionReductionScheduler() {
+  int bid_x = 80;
```

Comment:
const ? (or constexpr if you're comfortable with it, since these are really compile time constants)